library.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "libusb_stm32",
+	"description": "Lightweight USB Device Stack",
+	"version": "0.1.0",
+	"keywords": "stm32, usb",
+	"authors": {
+		"name": "Dmitry Filimonchuk",
+		"email": "dmitrystu@gmail.com"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/dmitrystu/libusb_stm32.git"
+	},
+	"frameworks": [ "cmsis", "stm32cube" ],
+	"platforms": [ "ststm32" ],
+	"headers": ["stm32_compat.h", "usb.h"],
```

Comment:
Why specify only these 2 headers? AFAIK all headers should be accessible from the consumer project.

Reply:
these are the ones that platformio will search for in library database if you happened not to add the relevant dependency.

regardless of this list, all header files are available to user one they add the library.

Reply:
adding all of the headers to this list would pollute the online header lookup mechanism.

Reply:
Oh I see, perhaps the docs here are incomplete? https://docs.platformio.org/en/latest//manifests/library-json/fields/headers.html